python/ray/serve/tests/test_autoscaling_metrics.py

```diff
@@ -99,6 +111,7 @@ def last_timestamp_value():
 
 if __name__ == "__main__":
     import sys
+
```

Comment:
extra line

Reply:
I think the isort (rencetly added) is doing the magic.